test/test_integration.py

```diff
@@ -0,0 +1,114 @@
+# coding: utf-8
+
+#
+# Copyright 2014-2017 Groupon, Inc.
+# Copyright 2014-2017 The Billing Project, LLC
+#
+# The Billing Project, LLC licenses this file to you under the Apache License, version 2.0
+# (the "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at:
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+"""
+    Kill Bill
+
+    Kill Bill is an open-source billing and payments platform  # noqa: E501
+
+    OpenAPI spec version: 1.0.0
+    
+    Generated by: https://github.com/swagger-api/swagger-codegen.git
+"""
+
+
+from __future__ import absolute_import
+
+import unittest
+
+import killbill
+from killbill.models.account import Account  # noqa: E501
+from killbill.models.tenant import Tenant  # noqa: E501
+from killbill.models.payment_method import PaymentMethod  # noqa: E501
+from killbill.models.subscription import Subscription  # noqa: E501
+from killbill.models.invoice_dry_run import InvoiceDryRun  # noqa: E501
+from killbill.rest import ApiException
+
+
+
+class TestIntegration(unittest.TestCase):
+    """Account unit test stubs"""
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def test_integration(self):
+
+        # Create tenant
+        # TODO WORKING WITH ISSUES*! [ValueError: Invalid value for `api_key`, must not be `None`]
+        tenant = killbill.api.TenantApi()
+        body = Tenant(api_key='bob_5', api_secret='lazar_5')
+        tenant.create_tenant(body, 'test')
+
+        # Get tenant
+        # TODO NOT WORKING! [ApiException: (401) Reason: Unauthorized] or [ValueError: Invalid value for `api_secret`, must not be `None`]
+        body = Tenant(api_key='bob')
+        tenant.get_tenant_by_api_key(body)
+
+        # Upload XML catalog/Fetch XML catalog
+        # TODO NOT WORKING! [ApiException: (404) Reason: Not Found]
+        catalog = killbill.api.CatalogApi()
+        xml_catalog = open("/home/jfrodriguez/Documents/KillBill-Docs/SpyCarBasic.xml", "r+").read()
+        catalog.upload_catalog_xml(xml_catalog, 'test', 'bob', 'lazar')
+
+        # Create account
+        # WORKING!
+        account = killbill.api.AccountApi()
+        body = Account(name='Johny', currency='USD', state='CA', country='USA')
+        account.create_account(body, 'test', 'bob', 'lazar')
+
+        # Get account
+        # TODO NOT WORKING! [AttributeError: 'module' object has no attribute 'Str']
+        account = account.get_account_by_key('test_external_key', 'bob', 'lazar')
```

Comment:
Account with `test_external_key` doesn't exist, so this will throw `org.killbill.billing.account.api.AccountApiException`. Specify the external key instead when creating the account:

```
account = killbill.api.AccountApi()
body = Account(name='Johny', external_key='account_key', currency='USD', state='CA', country='USA')
account.create_account(body, 'test', 'bob', 'lazar')
account_data = account.get_account_by_key(body.external_key, 'bob', 'lazar')
account = account.get_account(account_data.account_id, 'bob', 'lazar')
```